### DISCUSSION TITLE
New ATEM Constellation

### DISCUSSION THREAD

**Discussion by randallpacker** [danielbuechele/atemOSC#270] [General]
I am thinking about purchasing a new ATEM Constellation HD, it's light years beyond the ATEM HD. Curious to know if atemOSC functionality will apply to all the basics, plus the added upstream keyers, as well as the animations in the media pool. 

Many thanks!! 

Randall

**Comment by Distortman** [comment]
Hello, I'm wondering the same thing. I could be wrong, but I believe ATEM Constellations require at least firmware 8.7 and it looks like atemOSC can only go up to 8.6.4. I would really appreciate if anyone could give some insight on this. Thank you!

**Comment by randallpacker** [comment]
I am using ATEM Software Control 8.8 and atemOSC is working fine.

**Comment by SteffeyDev** [comment]
I'm curious what you find when tested

**Comment by randallpacker** [comment]
I just ordered it today so I'll definitely let you know.

**Comment by salmonerosee** [comment]
Do anybody know if the ATEM Constellation HD works fine with atem OSC?
I would like to know if it is possible to switch the sdi output via OSC, with "/atem/aux/1 1" or "/atem/aux/2 5" so that I can select which sdi output has to switch to which source.
Thank you!
